src/wallet/rpcwallet.cpp

```diff
@@ -2589,14 +2589,15 @@ static UniValue loadwallet(const JSONRPCRequest& request)
 
 static UniValue createwallet(const JSONRPCRequest& request)
 {
-    if (request.fHelp || request.params.size() < 1 || request.params.size() > 3) {
+    if (request.fHelp || request.params.size() < 1 || request.params.size() > 4) {
         throw std::runtime_error(
             RPCHelpMan{"createwallet",
                 "\nCreates and loads a new wallet.\n",
                 {
                     {"wallet_name", RPCArg::Type::STR, /* opt */ false, /* default_val */ "", "The name for the new wallet. If this is a path, the wallet will be created at the path location."},
                     {"disable_private_keys", RPCArg::Type::BOOL, /* opt */ true, /* default_val */ "false", "Disable the possibility of private keys (only watchonlys are possible in this mode)."},
                     {"blank", RPCArg::Type::BOOL, /* opt */ true, /* default_val */ "false", "Create a blank wallet. A blank wallet has no keys or HD seed. One can be set using sethdseed."},
+                    {"passphrase", RPCArg::Type::STR, /* opt */ true, /* default_val */ "", "Encrypt the wallet with this passphrase. The empty string (\"\") will result in an unencrypted wallet."},
```

Comment:
```suggestion
                    {"passphrase", RPCArg::Type::STR, /* default */ "''", "Encrypt the wallet with this passphrase. The empty string (\"\") will result in an unencrypted wallet."},
```

Mention that the default value is the empty string `"\"\""` or `"''"`.

Reply:
There really isn't a default value as the default is that there is no password and the wallet is unencrypted. So I've changed this to be `RPCArg::Optional::OMITTED`.